### ISSUE TITLE
Roles is missing from the peers json file

### ISSUE THREAD

**Issue by Wayonb** [nemtech/symbol-bootstrap#229]
**Describe the bug**
Creating a private bootstrap fails due to missing roles properties in the peers json files.

**To Reproduce**
To reproduce the behavior:
symbol-bootstrap start -p bootstrap --noPassword

**Logs**
````
peer-node-1          | 2021-04-04 21:20:11.428956 0x00007fc4c2d6aac0: <error> (config::PeersConfiguration.cpp@52) Throwing exception: required property 'roles' was not found in json 
db-0                 | {"t":{"$date":"2021-04-04T21:20:12.092+00:00"},"s":"I",  "c":"STORAGE",  "id":20320,   "ctx":"initandlisten","msg":"createCollection","attr":{"namespace":"local.startup_log","uuidDisposition":"generated","uuid":{"uuid":{"$uuid":"6cf0d124-6fec-4e65-9bb0-c26a7959af57"}},"options":{"capped":true,"size":10485760}}}
peer-node-1          | 2021-04-04 21:20:11.429894 0x00007fc4c2d6aac0: <fatal> (process::ProcessMain.cpp@75) 
peer-node-1          | thread: server catapult
peer-node-1          | unhandled exception while running local node!
peer-node-1          | /catapult-src/src/catapult/config/PeersConfiguration.cpp(52): Throw in function auto catapult::config::{anonymous}::Get(const ptree&, const string&) [with T = std::__cxx11::basic_string<char>; boost::property_tree::ptree = boost::property_tree::basic_ptree<std::__cxx11::basic_string<char>, std::__cxx11::basic_string<char> >; std::string = std::__cxx11::basic_string<char>]
peer-node-1          | Dynamic exception type: boost::wrapexcept<catapult::catapult_error<std::runtime_error> >
peer-node-1          | std::exception::what: required property 'roles' was not found in json
````

**Comment by fboucquez** [comment]
Yeap, found it while doing the testnet testing...

Fix is in the testnet  branch, going to be merged and released soon

https://github.com/nemtech/symbol-bootstrap/pull/224/files#diff-706b977723c41005d42a3e2e3206d0ebcfdafcb4ccda8e6d1ae3143f59d2491dR433

**Comment by fboucquez** [comment]
Fixed in 1.0.4
